inventory/byo/hosts.ose.example

```diff
@@ -167,6 +167,14 @@ openshift_release=v3.6
 # modify image streams to point at that registry by setting the following to true
 #openshift_examples_modify_imagestreams=true
 
+# If oreg_url points to a registry requiring authentication, provide the following:
+oreg_host: example.com:443
+registry_auth_user: some_user
+registry_auth_token: 'my-token'
```

Comment:
Yes, and if someone's registry is sitting at example.com/myregistry (ie, it's sitting behind a reverse proxy or something of that nature) then this will break for this purpose.  For example:  https://docs.docker.com/registry/recipes/nginx/#setting-things-up

I'm happy to change it, but I think we're introducing a bug, and I think it's bad practice to split variables rather than combine them.